src/exchange/ExchangeHomeScreen.test.tsx

```diff
@@ -40,6 +41,18 @@ describe('ExchangeHomeScreen', () => {
     expect(getByTestId('PriceChart')).toBeTruthy()
   })
 
+  it('renders the price chart using PriceHistoryChart and blockchain api when enabled', async () => {
+    jest.mocked(getFeatureGate).mockReturnValue(true)
```

Comment:
nit: can we reset the mock in a beforeEach or explicitly set in other tests so it doesn't affect them and only applies to this test?

Reply:
Fixed in 73561642b875cb292b7d136e1f82b5acae66c9b1!